docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -245,7 +245,7 @@ room.history.undo();
 
 ### Room.getOthers
 
-Gets all the other users in the Room.
+Gets all the other `User`s in the Room.
```

Comment:
I would love to add a dedicated page in the docs to document the `User` type (the return value of `room.getSelf()` and `room.getOthers()`. It has quite a few properties (like `connectionId`, `presence`, `id`, `isReadOnly`, etc.)

Those will be good to be able to explain and reference more clearly.